ScriptingExample/scripts/nope/Cell.zs

```diff
@@ -0,0 +1,30 @@
+public class Cell {
```

Comment:
Ah, you moved these files into the disabled folder?
We could also just remove them, or something lol